styles.css

```diff
@@ -1,504 +1,1051 @@
-body {
-  background: #e5e5e5;
-  margin: 0;
-  padding: 0;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-}
+@media only screen and (max-width: 767px) {
+  body {
+    background: #e5e5e5;
+    margin: 0;
+    padding: 0;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
 
-.header {
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-  justify-content: space-between;
-  padding-top: 25px;
-  width: 375px;
-}
+  .header {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+    padding-top: 25px;
+    width: 375px;
+  }
 
-.union {
-  padding-right: 20px;
-}
+  .union {
+    padding-right: 20px;
+  }
 
-#logo {
-  text-decoration: none;
-  width: 106px;
-  height: 20px;
-  font-family: "Poppins", sans-serif;
-  font-style: normal;
-  font-weight: 700;
-  font-size: 25px;
-  line-height: 20px;
+  .top_navbar {
+    display: none;
+  }
 
-  /* identical to box height, or 111% */
+  #logo {
+    text-decoration: none;
+    width: 106px;
+    height: 20px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 25px;
+    line-height: 20px;
 
-  display: flex;
-  align-items: center;
-  text-align: center;
-  padding-left: 17px;
+    /* identical to box height, or 111% */
 
-  /* Primary/B400 */
+    display: flex;
+    align-items: center;
+    text-align: center;
+    padding-left: 17px;
 
-  color: #6070ff;
-}
+    /* Primary/B400 */
 
-.main-section {
-  display: flex;
-  flex-direction: column;
-  background-image: url(./images/mobie.png);
-  width: 375px;
-  height: 616.15px;
-  left: 0;
-  top: 75px;
-  margin-bottom: 20px;
-  margin-top: 30px;
-}
+    color: #6070ff;
+  }
 
-.primary-text {
-  width: 317px;
-  height: 150px;
-  margin-top: 100px;
-  font-family: "Poppins", sans-serif;
-  font-style: normal;
-  font-weight: 800;
-  font-size: 40px;
-  line-height: 52px;
-  display: flex;
-  align-items: center;
-  padding-left: 26px;
-  color: #172b4d;
-}
+  .main-section {
+    display: flex;
+    flex-direction: column;
+    background-image: url(./images/mobie.png);
+    width: 375px;
+    height: 616.15px;
+    left: 0;
+    top: 75px;
+    margin-bottom: 20px;
+    margin-top: 30px;
+  }
```

Comment:
- [ ] You have done a great job showcasing the main section of your project :+1:. However, I suggest adding the bottom left-rounded corner to match the template. Additionally, aligning the social links with the other elements on your main page would enhance the overall appearance of your project.
- [ ] Kindly remember to write "Let's connect" in uppercase letters.

**Your design**

![Screenshot from 2023-03-31 18-18-53](https://user-images.githubusercontent.com/105117832/229249183-3733e305-72f2-48b1-aa40-49789fb09884.png)

**Figma**

![Screenshot from 2023-03-31 18-19-10](https://user-images.githubusercontent.com/105117832/229249226-3209d23c-fcfc-46b5-829f-0f55e5d4ea1a.png)